documentation/src/main/markdown/runningdetect/runningairgap.md

```diff
@@ -1,23 +1,23 @@
 
 ## Adding the [blackduck_signature_scanner_name] to your air gap archive
 
 To create an air gap archive that includes the [blackduck_signature_scanner_name], follow these steps:
 
-1. Unzip the [company_name] [solution_name] air gap archive to create the [company_name] [solution_name] air gap directory.
-1. Download the appropriate [blackduck_signature_scanner_name] zip file from your Black Duck instance (System > Tools > Legacy Downloads > Signature Scanner), and unzip it. This will create a directory with a name like scan.cli-x.y.z.
-1. Move that scan.cli-x.y.z directory to the top level of the [company_name] [solution_name] air gap directory.
-1. Zip the enhanced [company_name] [solution_name] air gap directory to create your enhanced air gap archive.
+1. Unzip the [detect_product_short] air gap archive to create the [detect_product_short] air gap directory.
+1. Download the appropriate [blackduck_signature_scanner_name] zip file from your [bd_product_short] instance (System > Tools > Legacy Downloads > Signature Scanner), and unzip it. This will create a directory with a name like scan.cli-x.y.z.
+1. Move that scan.cli-x.y.z directory to the top level of the [detect_product_short] air gap directory.
+1. Zip the enhanced [detect_product_short] air gap directory to create your enhanced air gap archive.
 
-When you later run [company_name] [solution_name] from the directory created by unzipping your enhanced air gap archive, set property detect.blackduck.signature.scanner.local.path to the path to the scan.cli-x.y.z directory in your enhanced air gap archive directory.
+When you later run [detect_product_short] from the directory created by unzipping your enhanced air gap archive, set property detect.blackduck.signature.scanner.local.path to the path to the scan.cli-x.y.z directory in your enhanced air gap archive directory.
 
 ## Preparing and running in air gap mode
 
-To prepare to run [company_name] [solution_name] in air gap mode, unzip the air gap archive to create the air gap directory.
+To prepare to run [detect_product_short] in air gap mode, unzip the air gap archive to create the air gap directory.
 Do not make changes to files in the air gap directory.
-Invoke the [company_name] [solution_name] .jar file from its original unzipped location at the top level of the air gap directory.
-For more information on invoking the .jar file, refer to [Running the Synopsys Detect .jar](../runningdetect/basics/runningjar.md).
+Invoke the [detect_product_short] .jar file from its original unzipped location at the top level of the air gap directory.
+For more information on invoking the .jar file, refer to [Running the Detect .jar](../runningdetect/basics/runningjar.md).
```

Comment:
Any reason for not using [detect_product_short] here?

Reply:
@andrian-sevastyanov Only due to thinking it might be crafty to "future proof" it by calling it just detect.jar ; )
Do we know yet what the new full name will be?

Reply:
Not 100% sure if that's what you're asking, but the file name of the JAR will indeed be `detect.jar`.

Reply:
My bad, I was looking at one comment and replying to another. 
In this case, when the bracketed variable is used within the existing brackets for the link title, the variable does not resolve.
i.e. ends up appearing in html as: _[Running the [detect_product_short] .jar]_